packages/core/src/components/select/select.scss

```diff
@@ -0,0 +1,92 @@
+@use '../../helpers';
+@use '../../mixins';
+@use '../icon/icon';
+
+@mixin Select() {
+  @include icon.Icon();
+
+  @if not mixins.includes('Select') {
+    @include _Select();
+  }
+}
+
+@mixin _Select() {
+  .ods-select {
+    align-items: center;
+    display: inline-flex;
+    position: relative;
+    width: helpers.space(50);
+
+    &.-borderless {
+      width: fit-content;
+    }
+  }
+
+  .ods-select-native {
+    @include mixins.as-text-field();
+    @include mixins.interactive();
+    @include with-padding(1px); // border width set on `as-text-field` mixin
+
+    appearance: none; // removes native arrow that opens the select
+
+    &.-empty {
+      color: helpers.color('content-placeholder');
+    }
+
+    + .ods-icon {
+      color: helpers.color('content-main');
+      pointer-events: none;
+      position: absolute;
+      right: helpers.space(2);
+    }
+
+    &.-invalid,
+    &.-touched:invalid,
+    &.-invalid + .ods-icon,
+    &.-touched:invalid + .ods-icon {
+      color: helpers.color('content-negative');
+    }
```

Comment:
💅🏻 [nitpick]
```suggestion
    &.-invalid,
    &.-touched:invalid {
      &,
      + .ods-icon {
        color: helpers.color('content-negative');
      }
    }
```

Reply:
Can't do because of linting 🙄

Reply:
Isn't it done in the very next rule? 🤔 

```scss
&:disabled {
      &,
      + .ods-icon {
        color: helpers.color('content-disabled');
      }
    }
```

Reply:
No, specificity in this one is more than in "disabled" - it's `.-touched` with `:invalid` where as here it's only a `:disabled`.

Reply:
Weird that it's treated differently 🤔 
Specificity doesn't change between both versions, only nesting does.
Oh well 🤷🏻‍♂️

Reply:
Yeah, it's a linting thing, we might reconsider increasing it.